src/App.js

```diff
@@ -55,6 +60,12 @@ const App = () => {
           <Route path="findPw" element={<FindPw />} />
         </Route>
 
+        <Route path="keyword/*" element={<KeywordPage />}>
+          <Route index element={<KeywordFilterBar />}></Route>
+          <Route path="create" element={<AddKeyword />}></Route>
+          <Route path="modify" element={<ModifyKeyword />}></Route>
+          <Route path="setting" element={<SettingKeyword />}></Route>
+        </Route>
```

Comment:
```suggestion
        <Route element = {<AuthorizedRoute/>}>
        <Route exact path="chat/*" element={<ChatPage/>} >
        ~~~
        </Route>
        <Route path="keyword/*" element={<KeywordPage />}>
          <Route index element={<KeywordFilterBar />}></Route>
          <Route path="create" element={<AddKeyword />}></Route>
          <Route path="modify" element={<ModifyKeyword />}></Route>
          <Route path="setting" element={<SettingKeyword />}></Route>
        </Route>
        </Route>
```
키워드 페이지는 로그인된 회원만 접속해야되는 아래 채팅페이지 처럼 로그인된 상태에서만 접속할 수 있도록 감싸주면 더 좋을 것 같습니다.

Reply:
넵! 감싸주도록 하겠습니다

Reply:
[faed626](https://github.com/BCSDLab/KOALA_FRONT_END/pull/47/commits/faed62688bafedc800505b35fb0fa771cd5b367c)

Reply:
[c637672](https://github.com/BCSDLab/KOALA_FRONT_END/pull/47/commits/c6376721af36aefa485d7e652a2a0bea71548634)

Reply:
[5d612b0](https://github.com/BCSDLab/KOALA_FRONT_END/pull/47/commits/5d612b07ae8edaa87ec66c7ecb1d9fcc88ebb466)